airbyte-integrations/connectors/source-chartmogul/unit_tests/test_streams.py

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from http import HTTPStatus
+
+import pytest
+from source_chartmogul.source import Activities, Customers
+
+
+@pytest.fixture
+def patch_base_class(mocker):
+    # Mock abstract methods to enable instantiating abstract class
+    mocker.patch.object(Customers, "__abstractmethods__", set())
+    mocker.patch.object(Activities, "__abstractmethods__", set())
+
+
+# Customer stream tests
```

Comment:
You can group testing of specific classes in test classes:
eg. 
```
class TestCustomers:
    def test_requests_params(self):
        stream = Customers()
```
It would avoid the awkward suffix naming for tests such as `test_request_params_activities`